src/index.js

```diff
@@ -8,15 +8,15 @@ const {
   utils
 } = require('cozy-konnector-libs')
 const request = requestFactory({
-  // the debug mode shows all the details about http request and responses. Very useful for
-  // debugging but very verbose. That is why it is commented out by default
-  // debug: true,
-  // activates [cheerio](https://cheerio.js.org/) parsing on each page
+  // The debug mode shows all the details about HTTP requests and responses. Very useful for
+  // debugging but very verbose. This is why it is set to false by default
+  debug: false,
```

Comment:
I am sorry @dohseven but I really prefer to comment/decomment this line when debugging.

Reply:
No worries @doubleface, I'll keep it then. You're right, it's faster to comment/uncomment it than to change the true/false value.